packages/fleet/lib/models/node_config_overrides.ts

```diff
@@ -0,0 +1,153 @@
+import type knex from 'knex';
+import type { Result } from '@nangohq/utils';
+import { Ok, Err } from '@nangohq/utils';
+import { FleetError } from '../utils/errors.js';
+import type { NodeConfigOverride } from '../types.js';
+import type { RoutingId } from '@nangohq/types';
+
+export const NODE_CONFIG_OVERRIDES_TABLE = 'node_config_overrides';
+
+export interface DBNodeConfigOverride {
```

Comment:
Following your own comment :D
```suggestion
interface DBNodeConfigOverride {
```